core/trace/src/main/scala/org/typelevel/otel4s/trace/SpanContext.scala

```diff
@@ -43,10 +52,14 @@ trait SpanContext {
     */
   def spanIdHex: String
 
-  /** Returns the sampling strategy of this [[SpanContext]]. Indicates whether
-    * the span in this context is sampled.
+  /** Returns the [[TraceFlags]] associated with this [[SpanContext]].
     */
-  def samplingDecision: SamplingDecision
```

Comment:
where is `SamplingDecision` used at this point?

Reply:
Currently, the `SamplingDecision` is not used anywhere. However, it will be used by the `sdk-trace` module (#325).